### DISCUSSION TITLE
Setting table to full width

### DISCUSSION THREAD

**Discussion by scvn1** [filamentphp/filament#7027] [Help]
How would I set the width of resource tables so they expand to max amount of screen space? Right now they max out at 1216 px.

**Comment by Z3d0X** [comment]
https://filamentphp.com/docs/2.x/admin/appearance#changing-the-maximum-content-width

**Comment by gilr** [comment]
Seems to have been removed in V3, or maybe replaced...
max_content_width and maxContentWidth (e.g. "protected ?string $maxContentWidth = 'full'; in V2)" don't exist anymore in the documentation (or even in the code).
(I don't know if it is temporary or definitive, I didn't found any information about it in the upgrade guide)

**Reply by danharrin** [reply]
https://filamentphp.com/docs/3.x/panels/pages#customizing-the-maximum-content-width

**Reply by gilr** [reply]
👍 Thank you! I saw it but as it is in the "custom page", I was not sure if it could work for the panel (and I think I gave it a try but maybe in the wrong place in the resource file 😞 ). But in the List page, it works! (the information could be added in the panels/resources/listing-records help page). Thanks again!

**Comment by bomshteyn** [comment]
If you want to change it on all pages in the panel you can set the width in the `PanelProvider`

```PHP
return $panel
    ...
    ->maxContentWidth('full')
    ...
```
